lib/omniauth/strategies/gds.rb

```diff
@@ -16,6 +16,7 @@ class OmniAuth::Strategies::Gds < OmniAuth::Strategies::OAuth2
       user: user,
       permissions: user['permissions'],
       organisation_slug: user['organisation_slug'],
+      organisation_slug: user['organisation_content_id'],
```

Comment:
Looking at the change in signon, It's probably that the key is wrong - it should be `organisation_content_id`